### DISCUSSION TITLE
glb file changing inline styling

### DISCUSSION THREAD

**Discussion by MikePageDev** [google/model-viewer#4110] [Q&A]
Hi, I have an odd one. I have been supplied with a glb file. I added some UI made of 2 buttons that move the camera around (which works fine) and a hotspot that on click zooms in and opens a module. I had this all working fine. I was then sent a new glb and now the hotspot is set to hidden and the positioning has changed. The glb is the only thing that has changed. 

Is it possible for the glb the modify CSS attributes?

**Comment by danilokrulj** [comment]
I don't think the glb file can modify CSS attributes. Maybe you have same issue as here #4076 ? I saw something about a new hotspot positioning system but I haven't researched it yet.

**Reply by elalish** [reply]
I think it's probably not the same as #4076, but perhaps related. If you use absolute hotspots (`data-position` & `data-normal`), they will be pretty transferrable between updated GLBs. If you use dynamic hotspots (`data-surface`) they will likely need to be regenerated for the new model. Our editor now creates dynamic hotspots, so you may just need to load the new model and reselect the hotspot location.
